### PR TITLE
New outline for the onboarding chapter

### PR DESCRIPTION
**Pull request by rpaik**
@semioticrobotic, @quaid, @palante this is still in draft/outline stage.

### CODE REVIEW

**Comment by rpaik** [comment]
Should be ready for the first review. 

@palante I have a placeholder section for "mentorship". Please feel free to rename the section and/or move it to a different spot in the chapter.

File: onboarding.adoc
Comment (semioticrobotic) [inline onboarding.adoc]: Placeholder for eventual overview of chapter-to-come.

File: onboarding.adoc
Comment (semioticrobotic) [inline onboarding.adoc:41]: Added some notes for potential section on contributor pathways. I think this is an important facet of community onboarding.

**Review by semioticrobotic — COMMENTED** [review]
I made a few edits and additions, and will work more soon. Great start, @rpaik!

**Comment by rpaik** [comment]
@semioticrobotic appreciate your review and edits. Like all your suggestions 😄

**Comment by semioticrobotic** [comment]
Thanks, @rpaik. Hoping to add a bit more over the next few days.

File: onboarding.adoc
Comment (palante) [inline onboarding.adoc:156]: Great work so far everyone! 

Quick question. Since we're adding this mentorship section back and expanding on it, thoughts on the edit process 🤔 
- Should I branch of this, make edits, and keep it consistent with changes going on here?
- Or just commit directly to this branch/PR? 

Let me know if have a preference, thanks!

Comment (semioticrobotic) [reply onboarding.adoc:156]: @rpaik is more experienced with GitHub than I am and may have a stronger opinion on workflow. I, however, would recommend/suggest working directly on this branch, so all three of us are hacking in the same space.

Comment (palante) [reply onboarding.adoc:156]: Thanks for the thoughts! Didn't have sufficient permissions to push commits directly on this branch || existing PR, so opened this one on @rpaik 's fork to add back the previous/existing mentorship content (cc @rpaik )-- https://github.com/theopensourceway/guidebook/pull/176#discussion_r535819864 :D

Comment (semioticrobotic) [reply onboarding.adoc:156]: Gotcha. Thanks, @palante. @rpaik, what's your feeling on merging @palante's changes in your branch to return the prior mentoring content?

**Review by rpaik — COMMENTED** [review]
Thanks @palante & @semioticrobotic I merged the new contents into my branch.

In general, I believe contents should be written in 3rd person? I noticed that the mentoring section was written in the first person ("I/my")....

**Comment by semioticrobotic** [comment]
You're correct, @rpaik. The material will need to be edited for tone, voice, consistency, etc. Slowly working my way there. :smile:

**Comment by palante** [comment]
Thanks @rpaik @semioticrobotic ! 

Good to know about 3rd person -- originally put back existing content and helpful to know going forward!

It looks like I don't have _write access_  so I think that's why unable to commit directly the branch and had to make the PR 😬 (thanks for merging that in). Maybe as an experiment, @rpaik are you able to commit to directly to this existing PR by @quaid ? (https://github.com/theopensourceway/guidebook/pull/180)

If so, would be great to see how you did it! I tried a few things but the non write access seemed to prevent.

And also, there's one more PR for additional mentorship content, thanks!

https://github.com/rpaik/guidebook/pull/2

**Comment by rpaik** [comment]
@semioticrobotic & @quaid I'm still getting familar with GitHub after a few years at GitLab, but it looks like only maintainers can make direct edits to a branch?

Not sure if there's a way to streamline the process with multiple authors?

**Comment by rpaik** [comment]
@semioticrobotic @palante, I actually discussed this with @quaid and others during this week's stand-up meeting...

I think it makes sense for mentorship to have its own chapter. Even before [the latest PR](https://github.com/rpaik/guidebook/pull/2)👆 from @palante, there's a lot of content on mentorship and it doesn't seem right for the topic to be buried under the "onboarding" chapter. I do mention mentors in the onboarding resources section and I can add a mention of the mentorship chapter as a cross-reference. 

If you're OK with this, I think the next steps can be: 

1. @rpaik to remove the mentorship section from this chapter. 
2. @palante can open a new PR (with a new branch from master) to start a new chapter on mentorship using the contents that he already has👆 

WDYT?

**Comment by semioticrobotic** [comment]
Certainly fine by me, @rpaik, but I defer to @palante's decision, as the person most interested in authoring that material.

**Comment by palante** [comment]
Sounds good on my end as well! 😄

Tonight, I can make a new PR that has the new mentorship content and the existing mentorship content. 
I'll add you both and @quaid as reviewers as well (eg edit, improvements, etc). 

Thanks all!

**Comment by rpaik** [comment]
Thanks, both for a quick response 💯 

I think this will also help alleviate the problem that we encountered when multiple "non-maintainers" (who don't have edit rights to branches) are collaborating on a chapter 😆

**Comment by quaid** [comment]
This is good stuff, folks, thanks. I am assigning issue #159 to @palante, which is from my proposed chapter titled of "Creating a Culture of Mentorship". This goes in the "Growing Contributors" section.

@palante It's OK if the title ends up changing or morph. We don't have a file for it in the repo yet, you can include creating it as part of the pull request. The format should follow creating_a_culture_of_mentorship.adoc. You can get the standard header from some of the other files, the template is like this:

= Chapter Name
// Authors: Authoor Name <email@example.com>
// Updated: YYYY-MM-DD
// Versions: 2.0
// Status: DRAFT

**Comment by palante** [comment]
Thanks all, created new PR: https://github.com/theopensourceway/guidebook/pull/182

**Comment by quaid** [comment]
@rpaik @semioticrobotic I'd like to land this branch soon, primarily so I can get copyedit help from folks without having them use the PR review process. When you get to a natural point, let's land the branch for the development edit. Then we can see if more content is needed, or if it's ready to prepare and send for copyedit.

**Review by quaid — APPROVED** [review]
This looks very good, especially from the development edit perspective.

I'm +1 on merging this, waiting for @rpaik to confirm (so we're not messing with any work in progress he wants to get into the PR before it's merged). I'm going to move the associated issue to the development edit column, as we are at that point. That column may require rewrites and is a good place process-wise to add more material if Ray wants to land anything.

**Comment by rpaik** [comment]
@quaid thanks for your review 😃 

I made some minor edits mostly to fix typos, but yes please feel free to merge this!

**Pull request merged**
